Documentation/ceph-dashboard.md

```diff
@@ -14,27 +14,40 @@ and more. Rook makes it simple to enable the dashboard.
 
 ## Enable the Dashboard
 
-The [dashboard](http://docs.ceph.com/docs/luminous/mgr/dashboard/) can be enabled with settings in the cluster CRD. The cluster CRD must have the dashboard `enabled` setting set to `true`.
+The [dashboard](http://docs.ceph.com/docs/mimic/mgr/dashboard/) can be enabled with settings in the cluster CRD. The cluster CRD must have the dashboard `enabled` setting set to `true`.
```

Comment:
Just to clarify this PR does not only decouple the Ceph versions but also update the used version to (default) Mimic, right?

Reply:
Correct, the default in the examples is now for Mimic. If a cluster is upgraded from v0.8 it will stay on Luminous by default (or if the cephVersion is not found in the CRD).  Where there are differences in behavior between versions I attempted to document it, such as the dashboard here that requires http for luminous and https for mimic and newer.